client/components/overviewWidget/StyleSelector.jsx

```diff
@@ -7,20 +7,16 @@ class StyleSelector extends React.Component {
     super(props);
 
     this.state = {
-      product: null,
       styles: [],
       selectedStyle: ''
     };
   }
 
-  componentDidMount() {}
-
   componentDidUpdate(prevProps) {
     if (this.props.product !== prevProps.product) {
       axios.get(`/api/fec2/hrnyc/products/${this.props.product.id}/styles`)
```

Comment:
If there's time later this week, we should think about changing how the selected style is loaded into this comps state. Making an API call for each style switch might affect our app's performance.